test_runner/logic_test_util.py

```diff
@@ -49,7 +49,7 @@ def RunLogicTestOnSim(
       simctl_env_vars[_SIMCTL_ENV_VAR_PREFIX + key] = env_vars[key]
   simctl_env_vars['NSUnbufferedIO'] = 'YES'
   command = [
-      'xcrun', 'simctl', 'spawn', sim_id,
+      'xcrun', 'simctl', 'spawn', '-s', sim_id,
```

Comment:
The upstream change https://github.com/google/xctestrunner/pull/5 appears to be using `--standalone` as the flag; should we use that here for consistency?

Reply:
Yes. Addressed in 97dcead41b18765a49eb50be806af4688da10e80.